unison-src/transcripts/fix1926.output.md

```diff
@@ -25,7 +25,7 @@ sq = 2934892384
 
     1 | > 'sq
           ⧩
-          'sq
+          do sq
```

Comment:
I guess this is fine?

Reply:
I guess.

It's easy to tweak the `do` insertion. I think I know the place to change this.

What's happening is that the surrounding precedence is 0, so it's allowed to use `do`. It's also allowed to use `do` as the last argument to a function call.